server/game/cards/14.6-Atonement/FireAndOil.js

```diff
@@ -0,0 +1,32 @@
+const DrawCard = require('../../drawcard.js');
+const { CardTypes, AbilityTypes } = require('../../Constants');
+const AbilityDsl = require('../../abilitydsl');
+
+class FireAndOil extends DrawCard {
+    setupCardAbilities() {
+        this.grantedAbilityLimits = {};
```

Comment:
What are these for?

Reply:
I'm not sure, it's in a lot of other cards?